inyoka_theme_default/templates/portal/user_edit_groups.html

```diff
@@ -26,49 +26,9 @@
     <h3>{% trans %}Group memberships{% endtrans %}</h3>
   </div>
 
-  {% call macros.outer_form(csrf_token(), form, manually_rendered=True) %}
-    <table class="user_groups form-group">
-      <tr>
-        <th class="col-sm-5">{% trans %}Available groups{% endtrans %}</th>
-        <th class="col-sm-1"></th>
-        <th class="col-sm-5">{% trans %}Membership{% endtrans %}</th>
-      </tr>
-      <tr>
-        <td>
-          <select multiple="multiple" size="5" name="user_groups_not_joined" class="form-control">
-          </select>
-        </td>
-        <td class="actions">
-          <div class="center-block col-sm-1">
-            <button class="item_add btn-default">
-              <span alt="{% trans %}Add group{% endtrans %}" class="fa_icon-arrow-right"></span>
-            </button>
-            <button class="item_remove btn-default">
-              <span alt="{% trans %}Remove group{% endtrans %}" class="fa_icon-arrow-left"></span>
-            </button>
-          </div>
-        </td>
-        <td>
-          <select multiple="multiple" size="5" name="user_groups_joined" class="form-control">
-          </select>
-        </td>
-      </tr>
-    </table>
-
-    {{ macros.input(form.primary_group) }}
-
-  {% endcall %}
-{% endblock %}
-
-{% block scripts %}
-  <script src="{{ href('static', 'js', 'classy.js') }}"></script>
-  <script src="{{ href('static', 'js', 'UserGroupBox.js') }}"></script>
-  <script type="text/javascript">
-    /* <![CDATA[ */
-    var select_form = new GroupBox('#user_edit_box',
-      {{ joined_groups|sort|jsonencode }},
-      {{ not_joined_groups|sort|jsonencode }}
-    );
-    /* ]]> */
-  </script>
+   <form enctype="multipart/form-data" class="portal-user-groupsetting-form" method="post" action="">
```

Comment:
this line should™ do the same:

    {{ macros.outer_form(csrf_token(), form, bootstrap_class="portal-user-groupsetting-form") }}